local-running/local-config.yml

```diff
@@ -0,0 +1,45 @@
+# This config file is intended for use in development, when running the Verify
+# Service Provider as part of a locally running federation. It includes usages
+# of advanced configuration options which should not be necessary for relying
+# parties who are running against compliance tool, or the hub integration or
+# live environments.
+
+server:
+  type: simple
+  applicationContextPath: /
+  adminContextPath: /admin
+  connector:
+    type: http
+    port: ${VSP_PORT:-50400}
+
+logging:
+  level: ${LOG_LEVEL:-INFO}
+  appenders:
+    - type: console
+    - type: file
+      currentLogFilename: logs/verify-service-provider.log
+      archivedLogFilenamePattern: logs/verify-service-provider.log.%d.gz
+
+clockSkew: ${CLOCK_SKEW:-PT5s}
+
+serviceEntityIds: ${SERVICE_ENTITY_IDS:-["http://dev-rp.local/SAML2/MD"]}
+
+verifyHubConfiguration:
+  environment: ${VERIFY_ENVIRONMENT:-COMPLIANCE_TOOL}
+  ssoLocation: ${FRONTEND_URI}/SAML2/SSO
+  metadata:
+    trustStore:
+      path: ../verify-local-startup/data/pki/metadata.ts
+      password: marshmallow
```

Comment:
Even better - you can make this:
```
trustStore:
  type: encoded
  store: ${METADATA_TRUSTSTORE}
  password: ${METADATA_TRUSTSTORE_PASSWORD}
```
which avoids the `../verify-local-startup` reference

Reply:
Is the metadata truststore password env variable actually being set? I can't see it.